content/blog/posts/2022-09-30-conference-basics.md

```diff
@@ -0,0 +1,67 @@
+
+As I’m getting ready to head to my next conference, here are some things I’ll be keeping in mind I wanted to share:
+
+## What to Wear to a Conference
+
+I started my career in software development during the pandemic, so I got very used to living in sweatpants and other forms of (athelesure)[https://www.youtube.com/watch?v=CYRENWT8lz8].
+
+As the conference date approached, I realized I had not figured out what to wear the next few days. Would it be alright to go off media tropes of casually dressed developers, or should I play it safe and get myself some business attire? I asked for advice from more seasoned folks in the field and it turned out people wore a mix of both. One recommendation would be to have some comfortable shoes, since you'll likely spend a decent amount of time walking around the venue or standing around while chatting wtih others.
+
+## What to Bring
+
+### Are Business Cards Necessary?
+
+I saw several recommendations online to bring business cards and jot down notes on ones you get, since you might forget what you talked to them about. This seems like a good idea but if you don’t happen to have any as I did, I found it was easy enough to add people directly on LinkedIn. You could always type out a short message as you’re sending an invite too. This also cuts out the step of adding people afterwards. CascadiaJS had a QR code tied to our tickets to connect with other attendees quickly, and I wouldn't be surprised if other events have a similar feature too.
+
+### Your Favorite Notetaking Gear
+
+This may depend on your learning style, as some people absorb information best by listening intently while others like myself pay attention better while taking notes. If you’re taking a workshop, you may also want to bring your laptop so you can follow along to any live tutorials. - If you forget, chances are, someone tabling will be giving away a notepad or notebook
+
+## What to Do at a Conference
+
+### Attend Talks & Workshops
+
+Talks and workshops take up the bulk of a tech conference schedule, and for good reason - they give us a glimpse on interesting things others have learned or come across. This may be particular to the conference I attended, but in a nutshell, I learned how far web development has come, from tools that make our work easier and those that make the web experience richer and more expansive.
+
+As a newer developer, it also gave me a chance to assess what I know and how much I've come across in my journey so far in tech. For example, while listening in on a talk on "data as a frontend developer," I realized it wasn't as black a blackbox as I keep imagining it.
```

Comment:
```suggestion
As a newer developer, it also gave me a chance to assess what I know and how much I've come across in my journey so far in tech. For example, while listening in on a talk on "data as a frontend developer," I realized it wasn't quite as complex as I keep imagining it.
```

I might suggest changing this reference. I think "black box" is slightly frowned upon these days.